src/node_constants.cc

```diff
@@ -20,7 +20,9 @@
 #endif
 
 namespace node {
-
+#if HAVE_OPENSSL
+const char * DEFAULT_CIPHER_LIST = DEFAULT_CIPHER_LIST_CORE;
```

Comment:
Understood. Did it this way largely because of NODE_DEFINE_STRING_CONSTANT using the passed in argument name as the constant name. Can refactor that tho to separate it out (works for your other comment re: using camel case in the constants)